server_addon/maya/client/ayon_maya/plugins/create/create_ornatrix_cache.py

```diff
@@ -0,0 +1,75 @@
+from ayon_maya.api import (
+    lib,
+    plugin
+)
+from ayon_core.lib import BoolDef, NumberDef
+
+
+class CreateOxCache(plugin.MayaCreator):
+    """Output for procedural plugin nodes of Ornatrix """
+
+    identifier = "io.openpype.creators.maya.OxCache"
```

Comment:
This creator will need to have matching settings to allow being disabled/enabled per studio - likely even disabled by default since I don't think Ornatrix comes with Maya by default?